src/components/routes/Routes.js

```diff
@@ -3,6 +3,8 @@ import { Route, Switch } from 'react-router-dom';
 import { UserPage, RepoPage, HomePage } from '../pages';
 
 export const Routes = () => {
+  // TODO: we can consolidate the render callback
```

Comment:
do you mean remove the `return` word below? should we do that now?

Reply:
we keep the return. its hard to put comments in between parenthesis ( )